src/Orchard.Web/Modules/Orchard.Projections/Providers/Filters/ContentFieldsFilter.cs

```diff
@@ -75,7 +75,9 @@ public class ContentFieldsFilter : IFilterProvider {
             var relationship = fieldTypeEditor.GetFilterRelationship(propertyName.ToSafeName());
 
             // generate the predicate based on the editor which has been used
-            Action<IHqlExpressionFactory> predicate = fieldTypeEditor.GetFilterPredicate(context.State);
+            dynamic fullState = context.State;
```

Comment:
Why `dynamic` ? You can't access the `VersionScope` property directly?

Reply:
fieldTypeEditor.GetFilterPredicate (2 lines above) method use a dynamic parameter as input. I need that information within IFieldTypeEditor.GetFilterPredicate in order to return the right column (Value or LatestValue) to query.